Orange/statistics/util.py

```diff
@@ -231,3 +231,70 @@ def weighted_mean():
             np.zeros(X.shape[1]),
             nans,
             X.shape[0] - nans))
+
+
+def _sparse_has_zeros(x):
+    """ Check if sparse matrix contains any implicit zeros. """
+    return np.prod(x.shape) != x.data.size
+
+
+def _nan_min_max(x, axis=0, func=None):
+    if not sp.issparse(x):
+        return func(x, axis=axis)
+    elif axis is None:
+        extreme = func(x.data, axis=axis) if x.data.size else float('nan')
+        if _sparse_has_zeros(x):
+            extreme = func([0, extreme])
+        return extreme
+    else:
+        r = []
+        if axis == 0:
+            x = x.T
+
+        # TODO check & transform to correct format
```

Comment:
X is usually in `csr` and hence when one calls this with `axis=0` x becomes `csc` (due to transposing), which is isn't efficient for row slicing.